README.md

```diff
@@ -8,80 +8,22 @@ transactions between organizations. Splinter lets you combine blockchain-related
 technologies -- such as smart contracts and consensus engines -- to build a wide
 variety of architectural patterns.
 
-See [splinter.dev](https://www.splinter.dev/) for Splinter documentation,
-release notes, and community information.
-
- &hercon; [Features](#features-and-concepts) &hercon; [Building Splinter](#building-splinter) &hercon; [Demos](#demos) &hercon; [License](#license) &hercon;
-
-## Features and Concepts
-
-Splinter allows the same network to do two-party private communication,
-multi-party private communication, and network-wide multi-party shared state,
-all managed with consensus. A Splinter network enables multi-party or two-party
-private conversations between nodes using circuits and services.
-
-  - A _**node**_ is the foundational runtime that allows an organization to
-  participate in the network.
-
-  - A _**circuit**_ is a virtual network within the broader Splinter network
-  that safely and securely enforces privacy scope boundaries.
-
-  - A _**service**_ is an endpoint within a circuit that sends and receives
-  private messages.
-
-A Splinter application provides a set of distributed services that can
-communicate with each other across a Splinter circuit.
-
-![Splinter private circuits with shared state](assets/diagram-splinter-circuits+3companies.svg)
```

Comment:
The images that are referenced should also be removed.

Reply:
Deleted the three unused images.